CHANGELOG.md

```diff
@@ -589,11 +591,16 @@ await myHelper(context);
 
 - Beginning of alpha versioning.
 
-[unreleased]: https://github.com/coda/packs-sdk/compare/v1.4.0...HEAD
-[1.2.3]: https://github.com/coda/packs-sdk/compare/v1.0.1...v1.2.3
-[1.2.2]: https://github.com/coda/packs-sdk/compare/v1.0.1...v1.2.2
-[1.2.1]: https://github.com/coda/packs-sdk/compare/v1.0.1...v1.2.1
-[1.2.0]: https://github.com/coda/packs-sdk/compare/v1.0.1...v1.2.0
+[unreleased]: https://github.com/coda/packs-sdk/compare/v1.4.1...HEAD
```

Comment:
I sorted this to match the order of the example at https://keepachangelog.com/en/1.1.0/ and fixed many of the base comparison version

Reply:
I see the base comparison version added by release-it for this release was also wrong, which might explain why so many others were wrong. `release-it` picked v1.0.1 as the comparison base for v1.4.1 when it should have been v1.4.0

Reply:
I've fixed this manually. After the release I'll see if `release-it` can continue doing the correct thing in the future after this PR or if maybe we need some kind of a config or command line argument change

Reply:
I verified in another branch that after this PR `release-it` will be able to correctly update the changelog in the future